### PR TITLE
Adding global cluster support, IAM roles support

### PR DESCRIPTION
**Pull request by jamengual**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -324,3 +324,25 @@ variable "reader_dns_name" {
...
+}
+
+variable "source_region" {
+  type = string
+  description = "Source Region of primary cluster, needed when using encrypted storage and region replicas"
+  default = ""
+}
+
+variable "iam_roles" {
+  type = list
```
Comment (aknysh) [inline variables.tf]:
```suggestion
  type = list(string)
```

TF 0.12 syntax

Comment (jamengual) [reply variables.tf]: changed

**Pull request merged**